pkg/tagger/collectors/common.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/tmplvar"
 )
 
+// OrchestratorScopeEntityID defines the orchestrator scope entity ID
+const OrchestratorScopeEntityID = "c843937b-8912-452c-be85-f50bd04ecc0f"
```

Comment:
how did you choose this ID? 
can we use an ID more self explanatory?  like "orchestrator-host-scope-entity-id"

Reply:
if never we run the "agent tagger-list" it will be more easy to know what is it.

Reply:
yeah for sure, just want to make sure it does not collide with anything.